pkg/mesh_install/install.go

```diff
@@ -155,3 +157,80 @@ func (i *Installer) triggerInjectedSidecarReconciliation() {
 		}
 	}
 }
+
+type KindSortOrder []string
+
+var InstallOrder KindSortOrder = []string{
```

Comment:
You've created a type mask so you don't need to redeclare in the literal syntax
```suggestion
var InstallOrder = KindSortOrder{
```

Reply:
Also Doc string since it's exported